public/styles/shared/base.css

```
@@ -0,0 +1,15 @@
+@font-face {
+  font-family: "Noto Sans";
+  src: url("../../fonts/NotoSans-Regular.ttf");
+  font-display: swap;
+}
+
+@font-face {
+  font-family: "Edo SZ";
+  src: url("../../fonts/edosz.ttf");
+  font-display: block;
+}
+
+body {
+  background-color: #f02017;
+}
```

```suggestion
}

```

Add a newline at the end of the file to remove this complaint.

---

Added